packages/suite/src/views/wallet/anonymize/components/CoinjoinSessionDetail.tsx

```diff
@@ -34,6 +34,10 @@ export const CoinjoinSessionDetail = ({
     const minEstimatedTime = Math.floor(estimatedTime - timeBuffer);
     const maxMiningFeeValue = `${maxFee} sat/vB`;
 
+    const skippingChance = skipRounds
+        ? ((1 - skipRounds[0] / skipRounds[1]) * 100).toFixed(0)
```

Comment:
Math.round would be more accurate here than toFixed, I guess?

Reply:
It doesn't matter a lot, but since there is some conflict anyway, maybe you can change it?

Reply:
6e756d4f2